cmd/eksctl-anywhere/cmd/createcluster.go

```diff
@@ -185,8 +193,45 @@ func (cc *createClusterOptions) createCluster(cmd *cobra.Command, _ []string) er
 		},
 		ManagementCluster: cluster,
 		Provider:          deps.Provider,
+		CliConfig:         cliConfig,
 	}
 	createValidations := createvalidations.New(validationOpts)
 
 	return createCluster.Run(ctx, clusterSpec, createValidations, cc.forceClean)
 }
+
+func (cc *createClusterOptions) directoriesToMount(clusterSpec *cluster.Spec, cliConfig *config.CliConfig) ([]string, error) {
```

Comment:
seems like we never return error here, I think we can remove it and simplify the code :)